gui-libs/webp-pixbuf-loader/webp-pixbuf-loader-0.0.1.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit gnome2-utils meson multilib-minimal
+
+DESCRIPTION="WebP Image format GdkPixbuf loader"
+HOMEPAGE="https://github.com/aruiz/webp-pixbuf-loader"
+SRC_URI="https://github.com/aruiz/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux"
```

Comment:
did you test all those arches? again, the current ebuild has none at all.
```
Keywords for x11-libs/gdk-pixbuf-loader-webp:
               |                             |   u   |  
               | a   a     p         s a r   |   n   |  
               | m   r i   p   h m s p l i m | e u s | r
               | d a m a p c x p 6 3 a p s i | a s l | e
               | 6 r 6 6 p 6 8 p 8 9 r h c p | p e o | p
               | 4 m 4 4 c 4 6 a k 0 c a v s | i d t | o
---------------+-----------------------------+-------+-------
20160328234507 | o o o o o o o o o o o o o o | 6 o 0 | gentoo
```

Reply:
Fixed in v3. Added intersection between libwebp & gdk-pixbuf . Since it's just glue code.
I cannot test all arches, but this could be valid point to assume it will work on these arches, right?

Is it ok with you?

Reply:
@a17r ?